projects/components/src/code-viewer/code-viewer.component.ts

```diff
@@ -0,0 +1,110 @@
+import { ChangeDetectionStrategy, Component, Input, OnChanges, TemplateRef } from '@angular/core';
+import { Color, TypedSimpleChanges } from '@hypertrace/common';
+import { isEmpty, isNil } from 'lodash-es';
+import { DownloadFileMetadata } from '../download-file/download-file-metadata';
+
+@Component({
+  selector: 'ht-code-viewer',
```

Comment:
How is this different than snippet viewer? Please consolidate.

Reply:
It's possible all usages of it have been removed - if you would prefer to remove that in favor of this one, rather than updating it for the new use cases, that may be a fine option. We just don't want both hanging around.

Reply:
snippet viewer is very basic and we don't use it anymore since we started using the markdown-viewer. `snippet-viewer` is a subset of `ht-code-viewer` and eventually we could retire it.

Reply:
Removed snippet viewer and moved those configs here